src/plot_erpimage.jl

```diff
@@ -13,7 +10,7 @@ Plot an ERP image.
 ## Keyword Arguments
 - `erpblur` (Number, `10`): number indicating how much blur is applied to the image; using Gaussian blur of the ImageFiltering module.
 Non-Positive values deactivate the blur.
-- `sortix` (default: `nothing`): .
+- `sortix` (default: `nothing`): ???.
 - `sortvalues` (bool, `false`): parameter over which plot will be sorted. Using sortperm() of Base Julia. 
```

Comment:
sortvalues is not bool, but rather a vector to be sorted. - not sure what the default is